lib/beacon/loader/db_loader.ex

```diff
@@ -18,10 +19,24 @@ defmodule Beacon.Loader.DBLoader do
   end
 
   def load_from_db(site) do
+    Logger.info("[DBLoader] Loading site #{site} from db")
+    Logger.info("[DBLoader] memory: #{inspect(:erlang.memory())}")
```

Comment:
Might be better to emit telemetry events instead of logging this (or emit logs at `debug` level instead). I know with my various open source projects people have often requested control over what is logged and when, and `telemetry` really provides the best degree of control over this (as well as having other benefits).

Reply:
Yep that's planned. In this case it was a quick mechanism for debugging but it'll be removed before merging 👍🏻